src/web/frontend/redux/actions/fetchData.tsx

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import {response} from "express";
+
+// import {GET_DATA_INIT, GET_DATA_SUCCESS, GET_DATA_FAILURE} from '../constants'
+
+// export const getData = {
+//     init: () => ({
+//         type: GET_DATA_INIT,
+//     }),
+//     success: (request:any, response:any) => ({
+//         type: GET_DATA_SUCCESS,
+//         payload: {
+//             data: response,
+//         }
+//     }),
+//     failure: (error:any) => ({
+//         type: GET_DATA_FAILURE,
+//         payload: console.log('error: ', error )
+//     })
+// };
+
+function getFetch() {
+    return axios.get('api/room')
+}
+
+function setFetch(data:any) {
+    return {
+        type: 'GET_DATA',
```

Comment:
you have a special file with action names `constants.ts`
use the name from there

Reply:
and what is the difference between `GET_DATA` and `GET_DATA_INIT`